Configurations/auth.js

```diff
@@ -1,8 +1,8 @@
 const auth = {
 	discord: {
-		clientID: "",
-		clientSecret: "",
-		clientToken: "",
+		clientID: "381171142230147075",
+		clientSecret: "SUktHmXqgz_6-cYYSnc98kNMNQKTljI7",
+		clientToken: "MzgxMTcxMTQyMjMwMTQ3MDc1.DRLYlg.AvvNziQHWev06pZ10K6yKv0MqBw",

```

Comment:
**PLEASE REMOVE TOKEN**

Reply:
OH FUCK  
That's testing bot anyway, so